pretyping/detyping.ml

```diff
@@ -732,11 +729,13 @@ and detype_r d flags avoid env sigma t =
   match EConstr.kind sigma (collapse_appl sigma t) with
     | Rel n ->
       (try match lookup_name_of_rel n (fst env) with
-	 | Name id   -> GVar id
-	 | Anonymous -> GVar (!detype_anonymous n)
+         | Name id   -> GVar id
+         | Anonymous ->
+           let s = "_ANONYMOUS_REL_"^(string_of_int n) in
```

Comment:
The printers above do raise `Not_found` however here you just print, is that intended?

Reply:
Sorry, I'm not sure I see what you mean. Which printers above?

Reply:
Oh, but the `Not_found` (raised by the debug printer) is caught immediately. So the only difference is that this patch distinguishes unbound rels and rels to anonymous binders.